pkg/registry/core/service/storage/rest_test.go

```diff
@@ -279,6 +287,70 @@ func TestServiceRegistryCreate(t *testing.T) {
 	}
 }
 
+func TestServiceRegistryCreateDryRun(t *testing.T) {
+	storage, registry, server := NewTestREST(t, nil)
+	defer server.Terminate(t)
+
+	svc := &api.Service{
+		ObjectMeta: metav1.ObjectMeta{Name: "foo"},
+		Spec: api.ServiceSpec{
+			Selector:        map[string]string{"bar": "baz"},
+			SessionAffinity: api.ServiceAffinityNone,
+			Type:            api.ServiceTypeClusterIP,
+			ClusterIP:       "1.2.3.4",
+			Ports: []api.ServicePort{{
```

Comment:
Seems like we should also test requesting two items with the same node port? (Add port names to make that validate otherwise)